modules/administration/pages/icinga.adoc

```diff
@@ -151,7 +160,7 @@ systemctl enable apache2.service
 systemctl start apache2.service
 ----
 
-. Login to the WebGUI at: http://localhost/icinga.
+. Login to the WebGUI at: http://<client-hostname>/icinga.
 
 This concludes setup and initial configuration of Icinga.
```

Comment:
@Loquacity I am worried about the next section. It's about monitoring the server with Icinga, but AFAIK we don't support that anymore (we have prometheus for that).

Do we want to invest time on rewriting the whole section to describe how to monitor a simple version and then just point to the official doc?

Reply:
I wouldn't bother personally. It seems pretty unlikely that this will be published any time soon. Just comment it out.

Reply:
@juliogonzalez I agree with lana, lets comment this section out for now. We are promoting prometheus.